docs/docs/getting_started.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Getting Started
+tags: []
+---
+
+## A Quick Start
+
+The easiest way to use Noir is through `nargo` (inspired by Rust's `cargo`). You can use `noirup` to quickly install `nargo`:
+
+```bash
+curl -L noirup.dev | bash
+noirup
+```
+
+Since Noir is a backend-agnostic programming language for writing zero-knowledge proofs, we need to pair it with a proving backend.
+
+As an example, we can will use Aztec Lab's [Barretenberg backend](https://github.com/AztecProtocol/aztec-packages/tree/master/barretenberg). This is made easy by [`bbup`](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/bbup), which installs Barretenberg's [`bb` CLI tool](https://github.com/AztecProtocol/aztec-packages/tree/master/barretenberg):
+
+```bash
+curl -L bbup.dev | bash
+bbup
+```
+
+:::info
+
+_Always_ inspect what you run.
+
+- `noirup.dev` redirects to [this script](https://github.com/noir-lang/noirup/blob/main/install) and adds [this program](https://github.com/noir-lang/noirup/blob/main/noirup) to your path.
+- according to the [bbup documentation](https://github.com/AztecProtocol/aztec-packages/tree/master/barretenberg/bbup), `bbup.dev` redirects to [this script](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/bbup/install) which adds [this program](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/bbup/bbup.ts) globally, installing `node` if needed.
```

Comment:
Note that script currently doesn't exist.

Reply:
Yes it is pending on the other PR

Reply:
Yes, I'm flagging this as we shouldn't merge this change until that's merged.